tests/test_layers.py

```diff
@@ -71,38 +70,6 @@ def test_in_out():
         assert output.shape[2] == 2, "size not {}".format(output.shape[2])
         assert stop_tokens.shape[0] == 4
 
```

Comment:
Can you implement some tests for multi-speaker model? At least running it with a dummy input.

Reply:
I didn't remember that!
I just sent the commit that implements these tests!

Reply:
let me know if I need to do anything else!